Chess/game.h

```diff
@@ -33,7 +32,23 @@ public slots:
     void rollback_from_list();
     void make_move_from_list();
 
+public:
+    void cellAction_impl(QString);

```

Comment:
Back on OOP aproaches: cellAction and cellAction_impl belong to different interffaces - former is for external users (signals etc), later - for state. state should not have access to external interface, external users shoud not have acess to state interface.